product-service/ddl.sql

```diff
@@ -0,0 +1,14 @@
+create table if not exists products (
+	id uuid PRIMARY key default uuid_generate_v4(),
+    title text NOT null,
+    description text,
+    price integer,
+    image text
+);
+
+create table stocks (
+	id serial primary key,
+	product_id uuid UNIQUE,
+	count  integer,
+	foreign key ("product_id") references "products" ("id")
```

Comment:
Not to current task: Probably you will not be able to delete product row with respected stock available (you will need to add **on delete**)

Reply:
Sure, i'm going to improve it in future